src/Miew.js

```diff
@@ -2930,8 +2930,9 @@ Miew.prototype.resetPivot = function () {
     boundingBox.union(visual.getBoundaries().boundingBox);
   });
 
-  boundingBox.getCenter(this._gfx.pivot.position);
-  this._gfx.pivot.position.negate();
+  const center = new THREE.Vector3();
```

Comment:
Hi, I  have a question about new object creation. How often is the function called? Could we avoid using of `new` every time?

Reply:
Thanks for your comment, done